discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -36,6 +36,12 @@ def list_nodes_with_name_filter(self, name_filter):
         logging.info("Found domains %s", nodes)
         return nodes
 
+    def list_networks(self):
```

Comment:
don't it need to be part of interface?

Reply:
maybe even network controller?

Reply:
But this is calling libvirt functionatlity

Reply:
still think it must be part of interface

Reply:
why do we need another controller? its libvirt